org.metaborg.spoofax.core/src/main/java/org/metaborg/spoofax/core/syntax/JSGLR1I.java

```diff
@@ -112,16 +114,20 @@ public SGLRParseResult actuallyParse(String text, @Nullable String filename,
         } else {
             disambiguator.setHeuristicFilters(false);
         }
-        
+
         String startSymbol = getOrDefaultStartSymbol(parserConfig);
-        try {
-            return parser.parse(text, filename, startSymbol);
+        SGLRParseResult parseResult;
+        ret: try {
```

Comment:
This should be refactored into a separate function, labeled breaks are confusing.

Reply:
Done